drivers/s3.go

```diff
@@ -41,39 +41,41 @@ type s3OS struct {
 }
 
 type s3Session struct {
-	host           string
-	key            string
-	policy         string
-	signature      string
-	xAmzCredential string
-	xAmzDate       string
-	driver         *s3OS
+	host        string
+	key         string
+	policy      string
+	signature   string
+	credential  string
+	xAmzDate    string
+	storageType net.OSInfo_StorageType
+	fields      map[string]string
 }
 
 // S3BUCKET s3 bucket owned by this node
 var S3BUCKET string
 
-// S3REGION region of s3 bucket owned by this node
-var S3REGION string
-
 func s3Host(bucket string) string {
 	return fmt.Sprintf("https://%s.s3.amazonaws.com", bucket)
 }
 
 // IsOwnStorageS3 returns true if turi points to S3 bucket owned by this node
 func IsOwnStorageS3(uri string) bool {
-	return strings.HasPrefix(uri, s3Host(S3BUCKET))
+	return strings.HasPrefix(uri, s3Host(S3BUCKET)) ||
+		strings.HasPrefix(uri, gsHost(S3BUCKET))
```

Comment:
Some preliminary testing results:

The `S3BUCKET` variable is never set with google storage, so this will always returns false for Google. Probably need to set the variable (or an equivalent) in livepeer.go.

Also, now that I'm re-reading this, should the google check be put into `drivers.IsOwnExternal` instead of S3?

https://github.com/livepeer/go-livepeer/blob/61071f711fe219daa20615381444f591f21b37b9/drivers/drivers.go#L49-L51

Reply:
good point, I've separated them

Reply:
S3BUCKET still needs to be set in `livepeer.go` (or a google specific var to hold the bucket)

Reply:
my mistake, fixed